rapid7_insightidr/plugin.spec.yaml

```diff
@@ -124,6 +124,188 @@ types:
       description: Indicates whether this threat has been published
       type: boolean
       required: false
+  link:
+    rel:
+      title: Relation
+      description: Relation
+      type: string
+    href:
+      title: Href
+      description: Href
+      type: string
+  eventData:
+    oaState:
+      title: OA State
+      type: string
+      description: OA state
+      required: false
+      lastAsSignatureAge:
```

Comment:
Indentation starting here and throughout the rest of `eventData` is off by one level